app/scripts/controllers/encryption-public-key.test.ts

```diff
@@ -214,14 +214,18 @@ describe('EncryptionPublicKeyController', () => {
   });
 
   describe('newRequestEncryptionPublicKey', () => {
+    // @ts-expect-error This function is missing from the Mocha type definitions
```

Comment:
do we need to add https://www.npmjs.com/package/@types/mocha-each ?

Reply:
+1, it doesn't look too large otherwise could extend with a custom definitions file.

Reply:
It looks like `@types/mocha-each` is a set of types for the `mocha-each` package: https://github.com/ryym/mocha-each#readme

We could probably add this package and replace our usage of `it.each` with this `forEach` utility. But adding that types package alone would not fix this type error.

Reply:
Oh darn, I see what you mean now. This isn't missing from the Mocha type definitions, it doesn't exist at all. This test is run with Jest.

Reply:
The comment above makes sense.
Jest `it.each` !== Mocha's `each` utility.

These tests are (probably) not ran in mocha, but in jest. So we don't actually need any additional mocha types. I'm not fussed, we can fix these false positives in a different PR, lets get this type coverage for missing tests in.

Reply:
Hmm OK. Yeah, I am tempted to at least adjust this comment to reflect reality better, but I am leaning towards merging this as-is and removing this completely in a follow-up PR that ensures these lint rules are applied correctly, to minimize the PR review churn.

Reply:
I'll do that. Apologies for the misleading comment, I'll have a PR up to fix that ASAP.

Reply:
:raised_hands: